examples/webgl_postprocessing_unreal_bloom_selective.html

```diff
@@ -143,50 +142,28 @@
 
 			const gui = new GUI();
 
-			gui.add( params, 'scene', [ 'Scene with Glow', 'Glow only', 'Scene only' ] ).onChange( function ( value ) {
-
-				switch ( value ) 	{
-
-					case 'Scene with Glow':
-						bloomComposer.renderToScreen = false;
-						break;
-					case 'Glow only':
-						bloomComposer.renderToScreen = true;
-						break;
-					case 'Scene only':
-						// nothing to do
-						break;
-
-				}
+			gui.add( params, 'exposure', 0.1, 2 ).onChange( function ( value ) {
```

Comment:
Personally, I would prefer to retain the separation of model parameters from tone mapping parameters in the GUI. (It is less confusing to users, IMO.) We have used folders for that.

I would try to be consistent and put exposure/tonemapping last in the GUI.

Reply:
So you mean we should make a separate GUI folder "tonemapping" that holds the exposure control and a separate folder for the bloom settings, right?